odl/discr/lp_discr.py

```diff
@@ -228,6 +234,12 @@ def exponent(self):
         """The exponent ``p`` in ``L^p``."""
         return self._exponent
 
+    @property
+    def vector_field_space(self):
+        """vector field space."""
+        real_space = self.astype(self._real_dtype)
+        return ProductSpace(real_space, self.ndim)
```

Comment:
Apparently we have a `real_space` property so this could read `ProductSpace(self.real_space, self.ndim)`